partitura/score.py

```diff
@@ -201,9 +203,6 @@ def key_signature_map(self):
 
             kss = np.array([(t0, fifths, mode), (tN, fifths, mode)])
 
-        elif len(kss) == 1:
```

Comment:
these lines were removed since this case is covered by the new `interp1d` utility in `partitura.utils.generic`.